admin_manual/groupware/out_of_office.rst

```diff
@@ -0,0 +1,36 @@
+=====================
+Out-of-office feature
+=====================
+
+.. versionadded:: 28.0.0
+
+The out-of-office feature allows users to schedule an out-of-office period including a status and
+message that is integrated with other apps such as Nextcloud Mail. Please refer to the user
+documentation for more information about the feature itself. It may be disabled globally by admins.
```

Comment:
Does a link exist for the user docs? You could insert it here.

Reply:
Not yet but we can only link inside the same documentation, AFAIK. So I can't cross-link to the user doc from the admin doc.

Do you know if that is possible?

Reply:
It's not. We can only use absolute links to the deployment host.